### ISSUE TITLE
Unable to securely load ancestry RF model

### ISSUE THREAD

**Issue by poneill** [broadinstitute/gnomad_methods#533]
Hi, I'm trying to load the gnomad PCA RF model described in the workflow posted here (https://gnomad.broadinstitute.org/news/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset/) and I'm running into a few issues with this snippet:

```
# Loading of the v2 RF model requires an older version of scikit-learn, this can be installed using pip install -U scikit-learn==0.21.3
path_to_gnomad_rf = "gnomad.r2.1.RF_fit.pkl"
with hl.hadoop_open(path_to_gnomad_rf, "rb") as f:
    fit = pickle.load(f)
```

1. scikit-learn 0.21.3 only offers wheels for python <= 3.7, so installing 0.21.3 requires either rolling back the interpreter as well or building from source.
2. When I load the model using 0.21.3, I get the following warning:

`UserWarning: Trying to unpickle estimator RandomForestClassifier from version 0.19.1 when using version 0.21.3. This might lead to breaking code or invalid results. Use at your own risk.`

(Rolling sklearn back even further to 0.19.1 is unfortunately an even gnarlier installation.)

One solution would be to export the RF model using a more robust format, e.g. onnx.  A compromise might be to expose a test set so users can check that the RF model is behaving as intended after deserialization.  Or is there an easier way to load the RF model?

Thanks in advance!

**Comment by jkgoodrich** [comment]
Thank you very much for your suggestion to convert our models to the ONNX format, this change will help several users that struggled with the same problem that you had.

We have converted both models, and by tomorrow the following files will be available for use:
```
gs://gcp-public-data--gnomad/release/2.1/pca/gnomad.r2.1.RF_fit.onnx
gs://gcp-public-data--gnomad/release/3.1/pca/gnomad.v3.1.RF_fit.onnx
```

In the meantime they can be found in our public requester pays bucket:
```
gs://gnomad-public-requester-pays/release/2.1/pca/gnomad.r2.1.RF_fit.onnx
gs://gnomad-public-requester-pays/release/3.1/pca/gnomad.v3.1.RF_fit.onnx
```

Soon we will have an update to the blog post with a link to an example notebook showing how to use them. Here is the [PR](https://github.com/broadinstitute/gnomad_qc/pull/366) for that notebook if useful to you in the meantime.


Thank you for posting your issue, and the suggested fix. 

Let us know if there are any other questions.

- gnomAD Production Team
